AutomatedTesting/Gem/PythonTests/scripting/Entity_HappyPath_AddScriptCanvasComponent.py

```diff
@@ -41,33 +40,30 @@ def Entity_HappyPath_AddScriptCanvasComponent():
     """
     from utils import TestHelper as helper
     from utils import Tracer
-    from editor_entity_utils import EditorEntity
+    from editor_python_test_tools.editor_entity_utils import EditorEntity
     import azlmbr.legacy.general as general
+    import azlmbr.math as math
+    from scripting_utils.scripting_constants import (SCRIPT_CANVAS_UI)
 
-    LEVEL_NAME = "tmp_level"
-    WAIT_TIME = 3.0  # SECONDS
+    TEST_ENTITY_NAME = "test_entity"
 
-    # 1) Create temp level
     general.idle_enable(True)
-    result = general.create_level_no_prompt(LEVEL_NAME, 128, 1, 512, True)
-    Report.critical_result(Tests.level_created, result == 0)
-    helper.wait_for_condition(lambda: general.get_current_level_name() == LEVEL_NAME, WAIT_TIME)
+
+    # 1) Create temp level
```

Comment:
NIT: Inaccurate comment. This is opening an existing level.

Reply:
updated the test step comment to reflect what we are actually doing at that step.